demo/handler.h

```diff
@@ -229,6 +237,18 @@ class Handler : public content_analysis::sdk::AgentEventHandler {
     std::cout << "  Filepath: " << file_path << std::endl;
     std::cout << "  Machine user: " << machine_user << std::endl;
     std::cout << "  Email: " << email << std::endl;
+    if (request.has_print_data() && !print_data_file_path_.empty()) {
+      std::cout << "  Print data saved to: " << print_data_file_path_
+                << std::endl;
+      using content_analysis::sdk::ContentAnalysisEvent;
+      auto print_data =
+          ContentAnalysisEvent::ScopedPrintHandle::Create(request);
```

Comment:
I wonder if there should be an `ScopedPrintHandle AgentEventHandler::GetPrintData()` method instead of a static `Create()`?  This could help solve the double-`ClouseHandle()` issue mentioned above as `GetPrintData()` could maybe clear the print data from the request as it passes ownership to `ScopedPrintHandle`.

Reply:
Maybe a `TakePrintData` later on, but for now I don't think it matters much since the partners might not like our interface and ask for a different one, maybe we leave these kind of refactors for later?